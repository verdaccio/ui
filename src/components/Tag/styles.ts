import styled from 'react-emotion';

export const Wrapper = styled('span')`
  && {
    vertical-align: middle;
    line-height: 22px;
    border-radius: 2px;
    color: #485a3e;
    background-color: #f3f4f2;
    padding: 0.22rem 0.4rem;
    margin: 8px 8px 0 0;
  }
`;
