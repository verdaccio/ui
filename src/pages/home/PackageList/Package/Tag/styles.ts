import styled from '@emotion/styled';

export const Wrapper = styled('span')({
  verticalAlign: 'middle',
  lineHeight: '22px',
  borderRadius: '2px',
  color: '#485a3e',
  backgroundColor: '#f3f4f2',
  padding: '0.22rem 0.4rem',
  margin: '8px 8px 0 0',
});
