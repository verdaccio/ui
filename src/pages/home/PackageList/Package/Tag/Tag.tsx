import React, { ReactNode } from 'react';

import { Wrapper } from './styles';

interface Props {
  children: ReactNode;
}

const Tag: React.FC<Props> = ({ children }) => <Wrapper>{children}</Wrapper>;

export default Tag;
