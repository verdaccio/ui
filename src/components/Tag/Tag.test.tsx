import React from 'react';
import { shallow } from 'enzyme';

import Tag from './Tag';

describe('<Tag /> component', () => {
  test('should render the component in default state', () => {
    const wrapper = shallow(
      <Tag>
        <span>{'I am a child'}</span>
      </Tag>
    );
    expect(wrapper.html()).toMatchSnapshot();
  });
});
