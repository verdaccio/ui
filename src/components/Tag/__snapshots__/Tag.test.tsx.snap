// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tag /> component should render the component in default state 1`] = `"<span class=\\"css-le0azu\\"><span>I am a child</span></span>"`;
